prober/handler.go

```diff
@@ -60,7 +56,9 @@ func Handler(w http.ResponseWriter, r *http.Request, c *config.Config, logger lo
 	if !ok {
 		http.Error(w, fmt.Sprintf("Unknown module %q", moduleName), http.StatusBadRequest)
 		level.Debug(logger).Log("msg", "Unknown module", "module", moduleName)
-		moduleUnknownCounter.Add(1)
+		if moduleUnknownCounter != nil {
```

Comment:
I kinda feel like we should enforce that a non-nil `prometheus.Counter` is passed to this.

Reply:
I did this way because we don't need it in the agent...but I can change it if is problematic.

Reply:
Makes sense.